cmd/packages.go

```diff
@@ -248,7 +250,7 @@ func packagesExecWorker(userInput string) <-chan error {
 		}
 		defer cleanup()
 
-		catalog, d, err := syft.CatalogPackages(src, appConfig.Package.Cataloger.ScopeOpt)
+		catalog, relationships, d, err := syft.CatalogPackages(src, appConfig.Package.Cataloger.ScopeOpt)
```

Comment:
I resisted on making this update here. I think this is better fit under https://github.com/anchore/syft/issues/554 
 and/or https://github.com/anchore/syft/issues/558 where we consider the units of work, what they produce, or what they modify. That is, since `syft.CatalogPackages` only generates a part of an SBOM today it wasn't clear if this should be modified to generate an entire SBOM (it seemed awkward from the power user command objects since then there would be multiple SBOM objects to deal with, which seem semantically wrong)